Marlin/dac_mcp4728.cpp

```diff
@@ -112,6 +112,18 @@ uint16_t mcp4728_getVout(uint8_t channel) {
 }
 */
 
+/* Returns DAC values as a 0-100 percentage of drive strength */
+uint16_t mcp4728_getDrvPct(uint8_t channel) {return (uint16_t)(.5+(((float)mcp4728_values[channel]*100)/DAC_STEPPER_MAX));}
+
+/* Recieves all Drive strengths as 0-100 percent values, updates DAC Values array and calls fastwrite to update the DAC */
+void mcp4728_setDrvPct(float xPct, float yPct, float zPct, float ePct) {
```

Comment:
With `driverPercent` as an array, this could take the single argument `uint16_t percent[XYZE]`. Note also that these values should be `uint16_t` instead of `float`. (The `mcp4728_values` array is `uint16_t`.)